packages/components/ng-ovh-sso-auth-modal-plugin/package.json

```diff
@@ -40,7 +40,8 @@
     "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/ng-ovh-sso-auth-modal-plugin' --include-dependencies -- yarn run dev:watch"
   },
   "dependencies": {
-    "lodash": "~4.17.11"
+    "lodash": "^4.17.15",
```

Comment:
bonus points : lodash could easily be removed too it's only used here (for a "set"...) https://github.com/ovh/manager/blob/master/packages/components/ng-ovh-sso-auth-modal-plugin/src/index.js#L28
one could say that since we only import a single function it's fine but in reality in the case of lodash it adds a lots of weight to the dist

Reply:
Good find !